R/styler.R

```diff
@@ -19,7 +19,7 @@ if (getRversion() >= "2.15.1") {
   utils::globalVariables(c(
```

Comment:
Once we have removed the tidyr and dplyr dependency, we can probably also refrain from declaring the variables below as globals, right?

Reply:
No you still need them here because they are still used with Non-Standard Evaluation in arrange.

Reply:
Ok, right. And `filter` also I guess. I was just wondering whether it would be desirable to get rid of NSE completely.